sub-packages/bionemo-geneformer/src/bionemo/geneformer/scripts/infer_geneformer.py

```diff
@@ -233,6 +236,12 @@ def get_parser():
         help="Micro-batch size. Global batch size is inferred from this.",
     )
 
+    parser.add_argument(
+        "--skip-unrecognized-vocab-in-dataset",
+        action="store_false",
+        help="Set to False to verify whether all gene identifers are in the user supplied tokenizer vocab. Defaults to True which means that any gene identifier not in the user supplied tokenizer vocab will be excluded.",
```

Comment:
I think this docstring is very confusing...it's got some implied double negatives. Usually, when `--some_flag` is attached in the CLI, it implies "some_flag" = true. In short, even though "store_false" is an argparse option, I suggest changing this arg to be "store_true" and modifying the help message as well.